src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -699,6 +700,45 @@ final void testPostPatronAccountByIdItemByItemIdRequests(
     logger.info("Test done");
   }
 
+  @ParameterizedTest
+  @MethodSource("itemRequestsParams")
+  final void testPostPatronAccountByItemNonPageRequests(
+    boolean isEcsTlrFeatureEnabledInTlr, boolean isEcsTlrFeatureEnabledInCirculation) {
+
+    ecsTlrFeatureEnabledInTlr = isEcsTlrFeatureEnabledInTlr;
+    ecsTlrFeatureEnabledInCirculation = isEcsTlrFeatureEnabledInCirculation;
+
+    logger.info("Testing creating a hold on an item for the specified user");
+
+    final Response r = given()
+        .log().all()
+        .header(tenantHeader)
+        .header(urlHeader)
+        .header(contentTypeHeader)
+        .body(readMockFile(MOCK_DATA_FOLDER + "/request_testPostPatronAccountByIdItemByItemIdHold.json"))
+        .pathParam("accountId", goodUserId)
+        .pathParam("itemId", checkedoutItemId)
+      .when()
+        .post(accountPath + itemPath + holdPath)
+      .then()
+        .log().all()
+        .contentType(ContentType.JSON)
+        .statusCode(201)
+        .extract().response();
+
+    final String body = r.getBody().asString();
+    final JsonObject json = new JsonObject(body);
+    String expectedHoldFile = isEcsTlrFeatureEnabledInTlr
+      ? "/response_testPostPatronAccountByIdItemByItemIdPage.json"
+      : "/response_testPostPatronAccountByIdItemByItemIdHold.json";
```

Comment:
We can extract expected file path into test parameters

Reply:
done